### DISCUSSION TITLE
Add localization strategy price for prices

### DISCUSSION THREAD

**Discussion by mrmokwa** [cyrilletuzi/vscode-angular-schematics#927] [Ideas]
I'd love help supporting this extension, but unfortunatelly for me, how lives is Brazil, ~US$50.00 it just too much money.
If we could have a localization strategy for pices, it would probably make more people from 3th world to subscribe.

**Comment by cyrilletuzi** [comment]
Hello @mrmokwa,

Thank you for your feedback.

To celebrate 1 million installations., the last v6.11.2 update will give you access to the current 50% off discount, which is the best offer overall.

Once the discount period has ended, I will investigate and probably enable the "Purchasing Power Parity" option of Gumroad (the service used to sell the licences), which should do what you ask.
